src/message/ssdp.rs

```diff
@@ -54,18 +57,23 @@ impl SSDPMessage {
     ///
     /// The host header field will be taken care of by the underlying library.
     pub fn send<A: ToSocketAddrs, C, S>(&self, connector: &mut C, dst_addr: A) -> SSDPResult<()>
```

Comment:
Since we now support IPv6, I noticed that when sending to an IPv6 address on any ```unicast``` method calls, ```all_local_connectors``` would also return IPv4 connectors which would panic when trying to parse the host [here](https://github.com/icorderi/ssdp-rs/blob/feature/ipv6/src/net/connector.rs#L48).

We can either detect that early (within the ```unicast```) which would involve handling the ```ToSocketAddrs``` boilerplate, or in the ```connect``` method itself and return an error. I am open to either, though detecting early within ```all_local_connectors``` would be preferable I think.